pyirf/simulations.py

```diff
@@ -224,3 +329,69 @@ def _viewcone_pdf_integral(viewcone_min, viewcone_max, fov_low, fov_high):
     if scalar:
         return np.squeeze(integral)
     return integral
+
+def _fov_lonlat_grid_overlap(self, bin_edges_lon, bin_edges_lat, radius, subpixels=20):
```

Comment:
Please add a docstring here although this is a private method

Reply:
Also: do I see it correctly that this function does not properly handle `viewcone_min > 0`?

Reply:
Yes you're right. That is of course unintentional, will fix asap. Out of curiosity: when and why is this ever the case?

Reply:
LST simulates diffuse gammas on a ring for use with the standard wobble observations.